src/main/java/seedu/address/model/event/EventTitleContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.event;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+
+/**
+ * Tests that an {@code Event}'s {@code EventTitle} matches any of the keywords given.
+ */
+public class EventTitleContainsKeywordsPredicate implements Predicate<Event> {
+    private final List<String> keywords;
+
+    public EventTitleContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
+    }
+
+    //TODO: Once EventTitle class is added, we need to change this to
+    //TODO: StringUtil.containsWordIgnoreCase(event.getEventTitle().toString(), kewyord));
+    @Override
+    public boolean test(Event event) {
+        return keywords.stream()
+                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(event.getEventTitle(), keyword));
+    }
```

Comment:
My EventTitle class will likely be merged before yours, so actually do you want to just replace this now 
```suggestion
    //TODO: StringUtil.containsWordIgnoreCase(event.getEventTitle().toString(), kewyord));
    @Override
    public boolean test(Event event) {
        return keywords.stream()
                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(event.getEventTitle().toString(), keyword));
    }
```